Extensions/Lighting/lightobstacleruntimebehavior.js

```diff
@@ -18,33 +18,31 @@ gdjs.LightObstaclesManager.getManager = function (runtimeScene) {
   return runtimeScene.lightObstaclesManager;
 };
 
-gdjs.LightObstaclesManager.prototype.addObstacle = function (obstacle) {
+gdjs.LightObstaclesManager.prototype.addObstacle = function(obstacle) {
   this._obstacleRBush.insert(obstacle);
 };
 
-gdjs.LightObstaclesManager.prototype.removeObstacle = function (obstacle) {
+gdjs.LightObstaclesManager.prototype.removeObstacle = function(obstacle) {
   this._obstacleRBush.remove(obstacle);
 };
 
-gdjs.LightObstaclesManager.prototype.getAllObstaclesAround = function (
+gdjs.LightObstaclesManager.prototype.getAllObstaclesAround = function(
   object,
   radius,
   result
 ) {
   // TODO: This would better be done using the object AABB (getAABB), as (`getCenterX`;`getCenterY`) point
   // is not necessarily in the middle of the object (for sprites for example).
-  var ow = object.getWidth();
-  var oh = object.getHeight();
-  var x = object.getDrawableX() + object.getCenterX();
-  var y = object.getDrawableY() + object.getCenterY();
+  var x = object.getX();
+  var y = object.getY();
 
   var searchArea = gdjs.staticObject(
-    gdjs.LightObstaclesManager.prototype.getAllPlatformsAround
+    gdjs.LightObstaclesManager.prototype.getAllObstaclesAround
   );
-  searchArea.minX = x - ow / 2 - radius;
-  searchArea.minY = y - oh / 2 - radius;
-  searchArea.maxX = x + ow / 2 + radius;
-  searchArea.maxY = y + oh / 2 + radius;
+  searchArea.minX = x;
+  searchArea.minY = y;
+  searchArea.maxX = x + 2 * radius;
+  searchArea.maxY = y + 2 * radius;
```

Comment:
Should this be:

```suggestion
  searchArea.minX = x - radius;
  searchArea.minY = y - radius;
  searchArea.maxX = x + radius;
  searchArea.maxY = y + radius;
```

? Because the light is in the center, and you must identify obstacles that are around it, in a square that is containing the light.

Reply:
Oh ok I understand that you made the choice of having the light x,y position to be the top-left.
I wonder if we should instead use the center of the light as the x, y position.

It sounds naturally for a light, because it's where the light is emitted :)